.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,28 @@
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: get version from tag
+        id: get_version
+        run: |
+            realversion="${GITHUB_REF/refs\/tags\//}"
+            realversion="${realversion//v/}"
+            echo "::set-output name=VERSION::$realversion"
```

Comment:
Isn't there already a built-in github action for this ?

Reply:
Maybe also - but I connected the two so with this.